xc7/utils/prjxray_create_edges.py

```diff
@@ -629,8 +641,8 @@ def connect_at(
             return
         elif self.pins and other_connector.tracks:
             assert self.pins.site_pin_direction == SitePinDirection.OUT
-            assert self.pins.x == loc.grid_x
-            assert self.pins.y == loc.grid_y
+            #assert self.pins.x == loc.grid_x
```

Comment:
I've identified the issue behind disabling this assertion, I'll submit a PR tomorrow once I've cleaned it up.